Contributors.md

```diff
@@ -13,3 +13,5 @@ Use the following template
 - Anmol Jain[Github: anmoljain12](https://github.com/anmoljain12)
 - Anonymous [Github: anonymous](https://github.com/anonymous)
 - Saurabh Patel [Github : srbhptl39](https://github.com/srbhptl39)
+- Ankit kumar [Github : PrajapatiAnkit](https://github.com/PrajapatiAnkit/)
+
```

Comment:
Try updating the pull request title from Updating list to something meaningful like 'Add Ankit Kumar to contributors'